src/Conversion/ONNXToKrnl/Math/Elementwise.cpp

```diff
@@ -2233,6 +2252,11 @@ struct ONNXElementwiseVariadicOpLowering
       SmallVector<IndexExpr, 4> ubs;
       create.krnlIE.getShapeAsDims(alloc, ubs);
 
+      // FIXME fix in the future
```

Comment:
Fix are fine, please states what needs to be fixed.

Reply:
Removed it, there is nothing needed to be fixed here